qml/MainScreen.qml

```diff
@@ -66,6 +68,8 @@ Item {
     property bool aboutToClose: false
     property bool aboutToMinimize: false
 
+    property alias nightstand: nightstandMode.active
```

Comment:
IIuc this is for watchfaces right? Could you add a comment above to explain why it's not used in this file and that it's meant to be a stable API for watchface developers ?

Reply:
I've added a comment to that effect.